src/app/containers/NewStoryPromo/PromoTypes/TopStoriesPromo.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Promo from '../Promo';
+
+const TopStoriesPromo = () => {
```

Comment:
So these promos could live next to the Optimo page like the one for topics here: https://github.com/bbc/simorgh/tree/latest/src/app/pages/TopicPage/TopicPromo